cmd/rpcfuzz/argfuzz/argfuzz.go

```diff
@@ -0,0 +1,225 @@
+package argfuzz
+
+import (
+	"encoding/hex"
+	"math/rand"
+	"reflect"
+	"strconv"
+
+	"github.com/google/gofuzz"
+)
+
+func MutateExecutor(arg []byte, c fuzz.Continue) []byte {
+	switch rand.Intn(10) {
+	case 0:
+		return AddRandomCharactersToStart(arg)
+	case 1:
+		return AddRandomCharactersToMiddle(arg)
+	case 2:
+		return AddRandomCharactersToEnd(arg)
+	case 3:
+		return DeleteRandomCharactersInStart(arg)
+	case 4:
+		return DeleteRandomCharactersInMiddle(arg)
+	case 5:
+		return DeleteRandomCharactersInEnd(arg)
+	case 6:
+		return DuplicateBytes(arg)
+	case 7:
+		return AddSpacesRandomly(arg)
+	case 8:
+		return []byte(c.RandString())
+	case 9:
+		return nil
+	default:
+		// Do nothing
+	}
+
+	return arg
+}
+
+func ByteMutator(arg []byte, c fuzz.Continue) []byte {
+	if arg == nil {
+		return arg
+	}
+
+	arg = MutateExecutor(arg, c)
+	// fitty-fitty chance of more mutations
+	if rand.Intn(2) == 0 && arg != nil {
+		return ByteMutator(arg, c)
+	}
+
+	return arg
+}
+
+func MutateRPCArgs(args *[]interface{}, c fuzz.Continue) {
+	for i, d := range *args {
+		if d == nil {
+			continue
+		}
```

Comment:
true...i just updated it so that if it's nil, it will be set to a random string then go through the regular mutator. I wonder what would be a better approach here, i.e. set it to a random value (int, string, etc.) then run it through the mutators?